.config/.env.example

```diff
@@ -15,4 +15,14 @@ LuisAPIHostNameFR=canadacentral
 # Project
 PROJECT=callback-bot-backend
 ENVIRONMENT=local
-TZ=America/Vancouver
\ No newline at end of file
+TZ=America/Vancouver
```

Comment:
My only question is why Vancouver here and not Ottawa. Might make sense to set to the Central and head of state time zone by default in the example. Small change.

Reply:
I do not think we are currently use TZ in our code right now. this is just an example in env.examplea as I copy it from other repo.